### ISSUE TITLE
Add ability to send Websocket pings

### ISSUE THREAD

**Issue by ChristianReese** [a85/POSTMan-Chrome-Extension#10956]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

It seems there isn't a way to send pings on a Websocket connection in Postman. Messages can be sent in text/JSON/Binary, but it would seem pings (WebSocket opcode 0x9 per RFC 6455) cannot be sent at this time.

### Describe the solution you'd like

Add a way to send pings on the connection. Perhaps as an option alongside text/JSON/binary, or as an advanced mode where the opcode can be specified?

In addition, this existing issue will also need to be implemented to view pings/pongs on the connection: https://github.com/postmanlabs/postman-app-support/issues/10500

**Comment by rulrok** [comment]
Just a note:

sometimes servers don't fully comply on maintaining the connection up using the spec ping opcode.

Thus, they work on a custom ping message frame, for example:

```json
{
    "op": "ping"
}
```

To be able to make such request to work as a ping would help as well

**Comment by achandrase-chwy** [comment]
If Postman receives a ping message from WebSocket Server, Postman (WebSocket Client) should send back a pong payload. Both payloads should be configurable. Postman should intercept each ping request matching configured ping payload using literal string equals and call send method with pong payload. This will be very useful to keep WebSocket connection alive so testing can keep going without having to reconnect again and subscribe to events.

**Comment by ernst77** [comment]
Is there any alternative to test ping/pong ?

**Comment by AviStudio** [comment]
The problem is still actual, as I today had some headache when my server destroyed ws connection if it don't receive a PONG from PostMan :(

**Comment by tfrancois** [comment]
I agree also that this ping/pong functionality is desperately needed in Postman.

**Comment by outranker** [comment]
We need a ping/pong mechanism in postman websockets

**Comment by Clinton-SalesMarker** [comment]
Seriously we need ping and pong in websockets in Postman.

**Comment by razvanphp** [comment]
For our use-case, Postman is not usable in this case, since the server disconnects us after 60 seconds of no-ping received.

Can we maybe contribute this feature to [postman-runtime](https://github.com/postmanlabs/postman-runtime) as hardcoded interval? For UI, I see Keep Alive in Settings tab for MQTT, should be very similar and trivial to add.
